Doc/Tutorial/chapter_entrez.tex

```diff
@@ -511,15 +511,16 @@ \section{ELink: Searching for related items in NCBI Entrez}
 %cont-doctest
 \begin{minted}{pycon}
 >>> len(record[0]["LinkSetDb"])
-7
```

Comment:
Good - I'd seen this while working on something else but hadn't got round to updating it.